kernel/kprobes.c

```diff
@@ -52,10 +53,36 @@ void kplist_del(struct kprobe *kp)
 	}
 }
 
+static void *kprobe_addr(struct kprobe *kp)
+{
+	void *addr = kp->addr;
+
+	/* Cannot use address and symbol name at the same time */
+	if ((kp->addr && kp->symbol_name) ||
```

Comment:
I think I'll need to make a script to check this kind of hard find coding style problem......